### DISCUSSION TITLE
Improvement for select with multiple modifier

### DISCUSSION THREAD

**Discussion by krnlde** [react-hook-form/react-hook-form#6677] [Ideas]
Hey Bill, I just saw this code piece

https://github.com/react-hook-form/react-hook-form/blob/3a4c79b61ec16b418fc0e3150ebf36a977944e6f/src/logic/getMultipleSelectValue.ts#L1-L6

and thought this could be improved by browser native properties

https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedOptions

Hope this is somehow useful.

Specifically with:
```js
Array.from(e.target.selectedOptions).map(({value}) => value)
```
you could skip the filter part.

Cheers mate :)

**Comment by bluebill1049** [comment]
hey @krnlde,

how are you? This is awesome! I love to save bytes, this change will be more performant and probably get us back to 8.2kb again. please send a PR, it's your contribution! ✨✌🏻 (if you are i can take over, and mention your name in the realse note)

Cheers
Bill

**Reply by krnlde** [reply]
I'm really fine thank you for asking :) Still using this library on a daily basis. How are you? How do you keep the issue count at around zero still :D that's still unbelievable. Keep it up!

**Reply by bluebill1049** [reply]
hahaha thanks 🙏 that's awesome to hear, great power comes with great responsibility. I am just doing my part, so everyone in the community can enjoy working with forms a bit more.

**Comment by krnlde** [comment]
It's not working in IE though. Is that a problem?

**Reply by bluebill1049** [reply]
nope, we have stopped stop IE11 from v7.

**Reply by bluebill1049** [reply]
<img width="1107" alt="Screen Shot 2021-09-30 at 8 58 10 pm" src="https://user-images.githubusercontent.com/10513364/135443132-7cab6e23-6020-45f9-afe3-2c8c89723786.png">
looks great.

**Comment by krnlde** [comment]
X-Ref: https://github.com/react-hook-form/react-hook-form/pull/6678
